hack/generators/release-controllers/content/art_publish_permissions.py

```diff
@@ -233,6 +233,14 @@ def add_art_publish(gendoc):
                         'namespace': f'ocp{config.get_suffix(arch, private)}'
                     }
                 })
+                gendoc.append({
+                    'apiVersion': 'image.openshift.io/v1',
+                    'kind': 'ImageStream',
+                    'metadata': {
+                        'name': f'{major_minor}-konflux-art-latest{config.get_suffix(arch, private)}',
+                        'namespace': f'ocp{config.get_suffix(arch, private)}'
+                    }
+                })
```

Comment:
Perhaps a check for "if major_minor > 4.17" might help the blast radius of this change (i.e. we don't need imagestreams from 4.0 and up).

Reply:
Completely understand if you don't want to.  Just trying to slim this down to what's really at stake.

Reply:
we would need it for all active OCP4 versions... < 4.12 is EOL, so maybe we can skip that? Not sure if a customer will suddenly ask us to support older versions though, like they did for 3.11

Reply:
I was thinking that konflux jobs would only be for more recent releases, going forward.  If they will be building all releases from greater than 4.12, then the block can be from there.

Reply:
Regardless, I don't really care.  Just don't want to be blamed for polluting the ocp namespace with imagestreams that will never be used.

Reply:
Thanks for the review. All of these will be deleted eventually, so I don't think it worth implementing any filtering logic.